docs/source/configuration/entity-caching.mdx

```diff
@@ -128,9 +128,157 @@ This entry contains an object with the `all` field to affect all subgraph reques
       "data": "key2"
     }
 }
+```
+
+### Invalidation
+
+Entity caching provides multiple ways to invalidate data:
+- an invalidation endpoint exposed by the router, that can receive invalidation requests from any authorized service out of band. This can be used by the subgraph to asynchronously trigger invalidation events, or by the platform operator to maintain the caching levels
+- invalidation requests can be sent through subgraph response extensions, allowing the subgraph to invalidate cached data right after a mutation
+
+Invalidation requests can be used to invalidate a large number of cached entries at once, matching either:
+- all cached entries for a specific subgraph
+- all cached entries for a specific type in a specific subgraph
+- all cached entries for a specific entity in a specific subgraph
```

Comment:
```suggestion
One invalidation request can invalidate multiple cached entries at once. It can invalidate:
- All cached entries for a specific subgraph
- All cached entries for a specific type in a specific subgraph
- All cached entries for a specific entity in a specific subgraph
```